src/workers/cdc.ts

```diff
@@ -225,6 +227,23 @@ const onAlertsChange = async (
   }
 };
 
+const onNotificationsChange = async (
```

Comment:
We discussed today this is will probably spam the system so it's better to send the message on the `readNotifications` mutation

Reply:
I agree. I think this should be covered [in this section](https://dailydotdev.atlassian.net/wiki/spaces/HAN/pages/378667009/PD-39+Notification+System#Reset-all-notifications-to-%E2%80%9CRead%E2%80%9D) (Reset all notifications to “Read” section) of the DR. Though for having a plus +1 in the unread, we can probably keep the `create` event here.

Reply:
Updated to create and readNotifications mutation